server/email/email-controller.js

```diff
@@ -0,0 +1,55 @@
+var mandrill = require('./email-service');
+var _ = require('underscore');
+
+module.exports = function(app, io){
+  var models = app.get('models');
+  var Gigs = models.Gig;
+  var UserGigs = models.UserGigs;
+  var Users = models.User;
+  var Locations = models.Location;
+
+  var emailController = {
+
+    sendEmails: function (req, res) {
+      var gigId = req.body.gigId;
+
+      Gigs.getGigInfo(gigId).then(function (gigInfo) {
```

Comment:
i see where you're coming from though, but i think that the optimization for sending emails off one by one is a small one since we won't be sending thousands of emails at once.

the models.sequelize.Promise.map way allows us to catch errors the result from getting the users from the db. if we do it the other way, the errors will go unnoticed in our catch block. thoughts?